Menu/Menu.js

```diff
@@ -1,10 +1,14 @@
 
+const MENU_OPEN_CLASS = 'menu--open';
```

Comment:
Hey nice work! This is an awesome use of a const to prevent errors and helps with the autocomplete in VSCode. Great